docs/exporting_data.md

```diff
@@ -120,11 +119,16 @@ bounding boxes to the image. When this is set to `True`, you must provide values
 Traceback (most recent call last):
   File "<stdin>", line 1, in <module>
   File "/workspace/armory/utils/export.py", line 205, in get_sample
-    raise TypeError("Both y_i and y_pred are None, but with_boxes is True")
+    raise TypeError("Both y_i and y_i_pred are None, but with_boxes is True")
 TypeError: Both y_i and y_pred are None, but with_boxes is True
 
 ```
-If you'd only like to include ground-truth boxes (or only predicted boxes), don't provide an arg for `y_i` or `y_i_pred`.
+If you'd like to include only ground-truth boxes (or only predicted boxes), provide an arg for only `y_i` (or only `y_i_pred`).
+
+For object detection, if `with_boxes` is `True`, the sample_exporter will also generate json files containing coco-formatted bounding box annotations.  These are saved in the scenario output directory as 
```

Comment:
I would delete the `if with_boxes is True` in the docs since it might make the user think they have to go set another field/kwarg somewhere. The object detection scenario code ensures that `with_boxes` is `True`

Reply:
That's fair, given that the preceding instructions are for `--interactive` I think that makes sense.

R.e. your question: I think a user should be able to simply export the raw image and only pass `x` to the call. If we always export boxes, then `y` becomes required. We used to decide whether to draw boxes based on whether a `y` is provided, but it feels more transparent to have an explicit kwarg.

Reply:
One thing I noticed, and this is unrelated to this PR so we can address in a separate PR, but when you're manually calling `export()` in `--interactive` mode you can no longer provide a `with_boxes` kwarg (or any kwarg not listed [here](https://github.com/twosixlabs/armory/blob/develop/armory/utils/export.py#L23)) due to [this](https://github.com/twosixlabs/armory/pull/1297/commits/00f9b761e560b489b144b6fa16a1f6ace47b689a) commit. So the function call in the docs actually will error

Reply:
So just to be clear, you feel ok about this as is?

Reply:
maybe one minor suggestion: how would you feel about changing `if with_boxes is True, the sample_exporter will ...` to `if with_boxes=True is passed to the export() call, the sample_exporter will...`

Reply:
see latest commit